site/content/en/docs/manual/advanced/ai-tools.md

```diff
@@ -1,155 +1,309 @@
+| High Resolution <br>Net (HRNet)                           | The model allows to get a mask for <br>an object using positive points (should <br>be left-clicked on the foreground), <br> and negative points(should be <br>right-clicked on the background, <br> if necessary). <br>It is recommended to run the model on GPU, <br>if possible.                                                | ![](/images/hrnet_example.gif) |
+| Inside-Outside-Guidance<br>(IOG)                          | The model uses a bounding box and <br>inside/outside points to create a mask. <br>First of all, you need to create a bounding<br> box, wrapping the object. <br>Then you need to use positive <br>and negative points to say the <br>model where is <br>a foreground, and where is a background.<br>Negative points are optional. | ![](/images/iog_example.gif)   |
 
-![](/images/iog_example.gif)
+<!--lint enable maximum-line-length-->
 
-## Detectors
+### Detectors
 
-Detectors are used to automatically annotate one frame. Supported DL models are suitable only for certain labels.
+Use detectors to automatically annotate one frame.
 
-- Before you start, click the `magic wand` on the controls sidebar and select the `Detectors` tab.
-  You need to match the labels of the DL model (left column) with the labels in your task (right column).
-  Then click `Annotate`.
+Supported DL models are suitable only for certain labels.
 
-  ![](/images/image187.jpg)
+To annotate with detectors, do the following:
 
-- Some of models supports attributes annotation (like facial emotions, for example: ``serverless/openvino/omz/intel/face-detection-0205``).
-  In this case you can also match attributes of the DL model with the attributes of a CVAT label.
+1. Click **Magic wand** ![Magic wand](/images/image189.jpg), and go to the **Detectors** tab.
+2. From the **Model** drop-down, select model (see [Detectors Models](#detectors-models)).
+3. From the left drop-down select the DL model label and match it to the right drop-down - the label of your task.
 
-  ![](/images/image187_1.jpg)
+   ![](/images/image187.jpg)
 
-- This action will automatically annotates one frame.
-  In the [Automatic annotation](/docs/manual/advanced/automatic-annotation/) section you can read
-  how to make automatic annotation of all frames.
+4. (Optional) If the model returns masks, and you
+   need to convert masks to polygon, use the **Convert masks to polygons** toggle.
+5. Click **Annotate**.
 
-### Mask RCNN
+Some of the models support attribute annotation like facial emotions,
+for example **Attributed face detection** (`serverless/openvino/omz/intel/face-detection-0205`).
 
-The model generates polygons for each instance of an object in the image.
+In this case, you can also match the attributes of the DL model with the attributes of a CVAT label.
 
-### Faster RCNN
+![](/images/image187_1.jpg)
```

Comment:
The image doesn't show actual attributes. Probably need to add corresponding attributes which are supported by the model and update the screenshot.